Tutorials/Python_Developer_Tutorials/Readme.md

```diff
@@ -0,0 +1,34 @@
+
+# Faraday Python Developer Tutorials
+
+This tutorial series is focused on introducing Faraday as a digital radio platform to program using Python. The reader will begin by learning fundamental transmissions of commands to both local and remote (using RF) devices. The tutorial series will conclude by building upon each skill and tool-set learned to develop a basic radio data  transmission program capable of transmitting data, text, etc... of arbitrary size from one Faraday to another through wireless transmissions.
```

Comment:
## Non-Blocking
* Spelling: http://www.dictionary.com/browse/toolset
* Wording "The tutorial series will conclude by building upon each skill and tool-set learned to develop a basic radio data  transmission program capable of transmitting data, text, etc... of arbitrary size from one Faraday to another through wireless transmissions." for example uses transmission or transmissions a lot which sounds weird.